osquery/events/linux/auditeventpublisher.cpp

```diff
@@ -178,6 +178,25 @@ void AuditEventPublisher::ProcessEvents(
       data.parent_process_id = static_cast<pid_t>(parent_process_id);
       audit_event.data = data;
 
+      std::uint64_t process_uid;
+      if (!GetIntegerFieldFromMap(
+              process_uid, audit_event_record.fields, "uid")) {
+        VLOG(1) << "Missing or invalid uid field in AUDIT_SYSCALL";
```

Comment:
Some of the other verbose messages have `Audit:` prepended, do you want to be consistent with those, or is there another differentiating feature?

Reply:
I initially wanted to prepend it to all messages. Do you like it, or do you think it is too verbose?